routingpy/routers/graphhopper.py

```diff
@@ -274,111 +258,107 @@ def directions(  # noqa: C901
 
         .. versionadded:: 0.3.0
            ``snap_prevention``, ``curb_side``, ``turn_costs`` parameters
+
+        .. versionchanged:: 1.2.0
+           Renamed `point_hint` to `point_hints`, `heading` to `headings`,
+           `snap_prevention` to `snap_preventions`, `curb_side` to `curbsides`,
+
+        .. versionadded:: 1.2.0
+           Added `custom_model` parameter
+
+        .. deprecated:: 1.2.0
+           Removed `weighting`, `block_area`, `avoid`, `turn_costs` parameters
```

Comment:
I've just assumed a new version number `1.2.0` here. Can be changed to something else

Reply:
Yep agreed